src/include/kdb.h

```diff
@@ -209,6 +209,16 @@ typedef struct _krb5_db_entry_new {
 
     krb5_principal        princ;                /* Length, data */
     krb5_tl_data        * tl_data;              /* Linked list */
+
+    /*
+     * Currently some parts of krb5 expect key_data to be sorted in reverse
+     * kvno order. Make sure this is sorted to prevent incorrect behavior from
+     * occuring.
+     *
+     * For more info please see here:
+     *
+     * https://github.com/krb5/krb5/pull/132#issuecomment-52522419
```

Comment:
I don't think we want to be linking to github PR comments from source code comments.  Also, I suggest cutting this comment down to "key_data must be sorted by kvno in descending order"; there is no need to hedge with "currently" or give a rationale.